koji/flatten-el8/koji-ansible/roles/grobisplitter/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: install python packages
+  package: name={{ item }} state=present
+  with_items:
+    - createrepo_c
+    - libmodulemd
+    - librepo
+    - python3-hawkey
+    - python3-librepo
+    - python3-gobject-base
+  tags:
+    - grobi
+
+- name: install python libraries from pip
+  pip:
+    name: repomd
+ 
+- name: make sure that /usr/local/bin exists
+  file: path=/usr/local/bin state=directory
+  tags:
+    - grobi
+
+- name: copy local/bin files
+  copy: src={{item}} dest=/usr/local/bin/ mode=0755
+  with_items: 
+    - splitter.py
+    - rhel8-split.sh
+  tags:
+    - grobi
+
+- name: daily cron job to split and merge repos
```

Comment:
It's odd to see cron items in a container.

Reply:
This is the role as taken from fedora-ansible.  It can probably be removed.  It's existence or removal is inconsequential in the grand scheme of things.